test_runner/random_test_runner/Readme.md

```diff
@@ -0,0 +1,213 @@
+# Random test runner
+
+Random test runner allows running randomly generated scenarios to test autoware autonomy implementation.
+
+| NOTE: Currently only `autoware.auto` (`awf/auto`) is supported. `AutowareArchitectureProposal` (`tier4/proposal`) will be supported in the future. |
+|----------------------------------------------------------------------------------------------------------------------------------------------------|
+
+## How to build
+
+### Autoware.auto
+
+It is assumed that you are running AutowareAuto in ADE environment with default paths. See [Autoware.Auto documentation](https://autowarefoundation.gitlab.io/autoware.auto/AutowareAuto/installation-ade.html) for instructions how to correctly set up Autoware.Auto and ADE.
+
+Inside Autoware.auto directory run/enter ade:
+```shell
+ade --rc .aderc-amd64-foxy-lgsvl start --update --enter
+```
+
+enter autoware directory, install required dependencies and build
+
+```shell
+cd <autoware_auto_directory_name>
+vcs import < autoware.auto.foxy.repos
+./tools/simulation/get_scenario_simulator_v2.sh
```

Comment:
This step clones `scenario_simulator` and checkouts it on commit id `13fd2047b2e394d6c84e24dcae6ffc520bc9fe6f`. Therefore it will not include the changes that introduce `random_test_runner`. There are two possible approaches:
* adding cone more command here: `checkout on commit XYZ`
* updating the commit id on AWF/AutowareAuto repository